README.md

```diff
@@ -950,7 +962,7 @@ simulate this behaviour using health checks and load balancing.
    docker start kongair-routes
    ```
 
-   Once the microservice is back up move on to the next section.
+   Once the microservice is back up move on to the next section, it may take a few seconds to detect the change by Kong Gateway.
```

Comment:
It's not recovered quickly, the traffic took about a minute to recover. Probably because Kong Gateway had cut the traffic to this upstream because of the previous failure status.